app/assets/stylesheets/application.scss

```diff
@@ -39,12 +39,6 @@ $hmrc-assets-path: "/check-tax-on-goods-you-bring-into-the-uk/assets/lib/hmrc-fr
   border: 2px solid #d4351c;
 }
 
-.govuk-warning-text__icon {
```

Comment:
Removed this as we don't need to position the icon as per [here](https://design-system.service.gov.uk/components/warning-text/)